apps/prairielearn/src/ee/pages/instructorAiGradingRuns/instructorAiGradingRuns.html.ts

```diff
@@ -0,0 +1,63 @@
+import { html } from '@prairielearn/html';
+
+import { PageLayout } from '../../../components/PageLayout.html.js';
+
+export function InstructorAIGradingRuns({ resLocals }: { resLocals: Record<string, any> }) {
+  return PageLayout({
+    resLocals,
+    pageTitle: resLocals.pageTitle,
+    navContext: {
+      type: 'instructor',
+      page: 'assessment',
+      subPage: 'manual_grading',
+    },
+    content: html` <div class="mb-3">
+        <a
+          href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+            .id}/manual_grading/assessment_question/${resLocals.assessment_question.id}"
+          class="btn btn-sm btn-primary"
+        >
+          <i class="fa fa-arrow-left" aria-hidden="true"></i>
+          Back to manual grading
+        </a>
+      </div>
+      <div id="ai-grading-card" class="card mb-5 mx-auto overflow-hidden" style="max-width: 700px">
+        <div class="card-body position-relative">
+          <h1 class="h3 text-center">AI Grading</h1>
+          <form id="ai-grading-form" name="ai-grading-form" method="POST">
+            <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+            <small class="form-text text-muted">
+              Grading student work using AI. More customizations will be enabled here in the future.
+              <br />
+              Select "Grade All" to grade all ungraded submissions. Select "Test Accuracy" to test
+              accuracy of AI using human-graded examples.</small
+            >
+            <div class="d-flex gap-2 mt-3">
+              <button
+                type="submit"
+                class="btn btn-success"
+                name="__action"
+                value="ai_grade_assessment"
+                data-toggle="tooltip"
+                data-placement="bottom"
+                title="Grade all ungraded submissions using AI"
```

Comment:
We've standardized on `data-bs-title` for tooltips. Ditto below.

```suggestion
                data-bs-title="Grade all ungraded submissions using AI"
```